### ISSUE TITLE
508-defect-3: "Track the status of your claim" link should be styled as an action link

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#55687]
## Point of contact

joshkimux

## Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

## Details

![image](https://user-images.githubusercontent.com/14154792/227579149-022b4d44-1ab8-4f91-88fa-f5ea6e424f8b.png)

On the confirmation page, the link "track the status of your claim" is styled as a button and not an action link.

## Reproduction steps

1. Log in and reach the confirmation page, right click the "track the status..." button and confirm it is actually a link
2. [On sketch, open up the design and confirm the link is styled as a button](https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/p/C0718005-83F4-4194-B62B-B63FC5F05282/canvas)

## Next steps
- [x] @eileen-coforma to confirm if this should be a button or an action link
- [ ] if confirmed as an action link, @rocio-desantiago or @Mottie to change the styling to the action link :heart

## Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [X] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by eileen-coforma** [comment]
@joshkimux So this is a weird one. You are right. Since it goes to a different page, it should be an action link. 

But, if we did this, the action link will be an outlier since the other confirmation pages use buttons for this action. We'll have to change this for the confirmation pages for 526, BDD, HLR, NoD. I can't check the 526ez submission right now, but I have a feeling that it's also a button on that confirmation page. Not sure how this passed for all the other forms.

**Comment by joshkimux** [comment]
@eileen-coforma ooph, that's definitely not cash money. [Confirmation pages don't seem to be a pattern in the VA design system](https://design.va.gov/about/whats-new), so I could write up an a11y discovery ticket to investigate other form's confirmation pages (and write up tickets if necessary). 

We can mark this as blocked until that discovery ticket is complete? Alternatively we can fix here and in the other spots we know and check them off as complete in said discovery ticket.

**Comment by joshkimux** [comment]
@eileen-coforma Perfect, I'll create a ticket for CAIA on this and block this off 💯

**Comment by joshkimux** [comment]
Update: created a CAIA ticket as a dependency!

**Comment by eileen-coforma** [comment]
I see a template for the [confirmation page structure](https://design.va.gov/templates/forms/confirmation) in the design system now. It's not specifically detailing the action of going to CST, but it does show action links for 'go back to VA.gov' which functions in the same way. 

I feel we can move forward with this, since CAIA has not responded about regulating confirmation pages.

**Comment by joshkimux** [comment]
Unblocking this, and will leave the CAIA ticket open so they don't forget about the other confirmation pages.

**Comment by davidakennedy** [comment]
I tested this, and see that the screen has been updated. An action link is present on the  Supplemental Claims confirmation page that leads to the Claim Status Tool. So I think that we can close this issue. @saderagsdale @eileen-coforma Let me know if I'm missing anything. Feel free to reopen.

![Red arrow pointing to an action link on the Supplemental Claims confirmation page that leads to the Claim Status Tool.](https://github.com/department-of-veterans-affairs/va.gov-team/assets/1473618/09bd730e-ce46-40b8-a563-c15700f67001)
